test/functional/services/remote/remote.ts

```diff
@@ -62,41 +35,22 @@ export async function RemoteProvider({ getService }: FtrProviderContext) {
   const { driver, consoleLog$ } = await initWebDriver(log, browserType, lifecycle, browserConfig);
   const caps = await driver.getCapabilities();
 
-  log.info(
-    `Remote initialized: ${caps.get('browserName')} ${caps.get(
-      'browserVersion'
-    )}, collectingCoverage=${collectCoverage}`
-  );
+  log.info(`Remote initialized: ${caps.get('browserName')} ${caps.get('browserVersion')}}`);
 
   if ([Browsers.Chrome, Browsers.ChromiumEdge].includes(browserType)) {
     log.info(
       `${browserType}driver version: ${caps.get(browserType)[`${browserType}driverVersion`]}`
     );
   }
 
-  consoleLog$
-    .pipe(
-      mergeMap((logEntry) => {
-        if (collectCoverage && logEntry.message.includes(coveragePrefix)) {
-          const [, coverageJsonBase64] = logEntry.message.split(coveragePrefix);
-          const coverageJson = Buffer.from(coverageJsonBase64, 'base64').toString('utf8');
-          writeCoverage(coverageJson);
-
-          // filter out this message
-          return [];
-        }
-
-        return [logEntry];
-      })
-    )
-    .subscribe({
-      next({ message, level }) {
-        const msg = message.replace(/\\n/g, '\n');
-        log[level === 'SEVERE' || level === 'error' ? 'warning' : 'debug'](
-          `browser[${level}] ${msg}`
-        );
-      },
-    });
+  consoleLog$.subscribe({
+    next({ message, level }) {
+      const msg = message.replace(/\\n/g, '\n');
+      log[level === 'SEVERE' || level === 'error' ? 'warning' : 'debug'](
+        `browser[${level}] ${msg}`
+      );
```

Comment:
Just an unimportant nit, but could be:
`log[level === 'SEVERE' || level === 'error' ? 'warning' : 'debug'](
        `browser[${level}] ${message.replace(/\\n/g, '\n')}`
      );